tools/ci_build/github/apple/c/onnxruntime-mobile-c.podspec.template

```diff
@@ -1,16 +1,18 @@
 Pod::Spec.new do |spec|
     spec.name                   = "onnxruntime-mobile-c"
-    spec.version                = "${ORT_VERSION}"
+    spec.version                = "@VERSION@"
     spec.authors                = { "ONNX Runtime" => "onnxruntime@microsoft.com" }
     spec.license                = { :type => "MIT" }
     spec.homepage               = "https://github.com/microsoft/onnxruntime"
```

Comment:
Do you want to replace this with 
    spec.license = { :type => "MIT", :file => 'LICENSE' }

Same for the podspec for objc?